manifests/agent.pp

```diff
@@ -81,6 +81,7 @@
   $ossec_rootcheck_frequency         = $wazuh::params_agent::ossec_rootcheck_frequency,
   $ossec_rootcheck_rootkit_files     = $wazuh::params_agent::ossec_rootcheck_rootkit_files,
   $ossec_rootcheck_rootkit_trojans   = $wazuh::params_agent::ossec_rootcheck_rootkit_trojans,
+  $ossec_rootcheck_system_audit      = $wazuh::params_agent::default_rootcheck_system_audit,
```

Comment:
No need to use an auxiliary variable `default_rootcheck_system_audit`.

Changed in 624daa7329176fc30fa571343f5aead28cc649d0 & 057f54eb9a4a6453dfbc6a99faec0a69f5aea988.

Thanks,